src/main/webapp/i18n/en/textAssessment.json

```diff
@@ -51,7 +51,14 @@
                 "dismissFeedbackConfirmation": "Please confirm that you want to delete this feedback!",
                 "feedbackCommentPlaceholder": "Enter your feedback comment here ...",
                 "impactWarning": "This feedback might affect <strong style=\"color:blue\">{{affectedSubmissionsCount}}</strong> other similar submission/s.",
-                "impactWarningTooltip": "The number of times that your feedback will be reused as an automatic suggestion in other submissions that were found to have a similar text block."
+                "impactWarningTooltip": "The number of times that your feedback will be reused as an automatic suggestion in other submissions that were found to have a similar text block.",
+                "automaticFeedbackInformationModal": {
+                    "heading": "Automatic Feedback Tracing",
+                    "introduction": "This automatic suggestion comes from a previously graded submission shown below.<br />The <span style='background-color: #d8e8f5'>highlighted feedback</span> has been reused.<br/><br/><i>Preview</i>",
```

Comment:
I am not sure about the `preview`. Its not a preview in the sense that you will see the full thing later on. I think no further heading would also be fine.

Reply:
It shows the original submission, right? then, we can call it like "Referenced Submission" instead of "Preview". Or  in the beginning of the modal, it already explains that it comes from previously graded submission, so we can remove the header for the box.

Reply:
hmm, I see. I guess removing it would make sense here 👍